requirements-common.txt

```diff
@@ -0,0 +1,2 @@
+ansible>=2.3
```

Comment:
```suggestion
ansible>=2.9
```

Reply:
`requirements-common` doesn't exist in FAM anymore, and we don't define a dependecy on ansible besides in the docs.